package.json

```diff
@@ -2489,6 +2489,316 @@
+															{
+																"key": "i",
+																"name": "Italic",
+																"type": "command",
+																"command": "latex-workshop.shortcut.mathit"
+															},
+															{
+																"key": "r",
+																"name": "Roman",
+																"type": "command",
+																"command": "latex-workshop.shortcut.mathrm"
+															},
+															{
+																"key": "t",
+																"name": "Terminal",
+																"type": "command",
+																"command": "latex-workshop.shortcut.mathtt"
+															}
+														]
+													}
+												]
+											},
+											{
+												"key": "g",
+												"name": "+Goto",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "e",
+														"name": "Navigate to matching begin/end pair",
+														"type": "command",
+														"command": "latex-workshop.navigate-envpair"
+													},
+													{
+														"key": "s",
+														"name": "Select the current environment name",
+														"type": "command",
+														"command": "latex-workshop.select-envname"
+													},
+													{
+														"key": "S",
+														"name": "Select the current environment content",
+														"type": "command",
+														"command": "latex-workshop.select-envcontent"
+													}
+												]
+											},
+											{
+												"key": "i",
+												"name": "+Insert",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "e",
+														"name": "Close current environment",
+														"type": "command",
+														"command": "latex-workshop.close-env"
+													},
+													{
+														"key": "i",
+														"name": "item",
+														"type": "command",
+														"command": "latex-workshop.shortcut.item"
+													},
+													{
+														"key": "w",
+														"name": "Surround/wrap selection with begin/end",
+														"type": "command",
+														"command": "latex-workshop.wrap-env"
+													}
+												]
+											},
+											{
+												"key": "l",
+												"name": "+Bibtex",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "a",
+														"name": "Align",
+														"type": "command",
+														"command": "latex-workshop.bibalign"
+													},
+													{
+														"key": "s",
+														"name": "Sort",
+														"type": "command",
+														"command": "latex-workshop.bibsort"
+													},
+													{
+														"key": "S",
+														"name": "Sort & Align",
+														"type": "command",
+														"command": "latex-workshop.bibalignsort"
+													}
+												]
+											},
+											{
+												"key": "v",
+												"name": "+View",
```

Comment:
Would this be more like `p` preview?

Reply:
We could rename this `p` preview. I chose `v` view since semantically it makes more sense. I know spacemacs has both `m v` and `m p *` variants. I'm fine with either.